pkg/persistence/queries.go

```diff
@@ -297,6 +346,22 @@ func registerDiagnosisKeys(db *sql.DB, appPubKey *[32]byte, keys []*pb.Temporary
 	if err = tx.Commit(); err != nil {
 		return err
 	}
+
+	if keysInserted > 0 {
+		event := Event{
+			Originator: originator,
+			Identifier: TEKUploaded,
+			DeviceType: Server,
+			Date: time.Now(),
+			// This is a safe conversion as we have a max of 28 keys as defined in the protobuf
+			// we should revisit if that changes
+			Count: int(keysInserted),
```

Comment:
This metric itself is not a problem. The log of the event `Logging Event {Identifier:TEKUploaded DeviceType:Server Date:2020-09-21 14:45:36.0493819 +0000 UTC m=+561.595133301 Count:14 Originator:thisisaverylongtoken}` regresses https://github.com/cds-snc/covid-alert-server/pull/280. We either need to filter that count, or we need to bring it back from a policy perspective.

Reply:
@Stephen-PM I'd like you to weigh in on this.

Reply:
Been removed